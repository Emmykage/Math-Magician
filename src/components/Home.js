import React from 'react';
import Header from './Header';

const Home = () => (
  <main className="container">
    <Header />

    <div className="page-layout">
      <h1>Welcome to Our page!</h1>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        Nam hendrerit nisi sed
        sollicitudin pellentesque. Nunc posuere purus rhoncus pulvinar aliquam.
        Ut aliquet tristique nisl vitae volutpat.
        Nulla aliquet porttitor venenatis.
        Donec a dui et dui fringilla consectetur id
        nec massa. Aliquam erat volutpat.
        Sed ut dui ut lacus dictum fermentum vel tincidunt neque.
        Sed sed lacinia lectus.
        Duis sit amet sodales felis. Duis nunc
        eros, mattis at dui ac, convallis semper risus.
        In adipiscing ultrices tellus, in susc
        ipit massa vehicula eu.”
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur
        adipiscing elit. Nam hendrerit
        nisi sed sollicitudin pellentesque. Nunc posuere purus rhoncus pulvinar aliquam.
        Ut aliquet tristique nisl vitae volutpat.
        Nulla aliquet porttitor venenatis.
        Donec a dui et dui fringilla consectetu
        r id nec massa.
        Aliquam erat volutpat. Sed ut dui ut lacus
        dictum fermentum vel tincidunt neque. Sed sed lacinia lectus.

        Duis sit amet sodales felis. Duis nunc ero
        s, mattis at dui ac, convallis semper risus.
        In adipiscing ultrices tellus, in suscipit
        massa vehicula eu.”
      </p>

    </div>
  </main>
);
export default Home;
 