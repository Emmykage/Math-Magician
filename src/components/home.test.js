import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Home from './Home';

test('Home renders a snapshot properly', () => {
  const tree = renderer.create(
    <Router>
            <Home />
        </Router>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
  
