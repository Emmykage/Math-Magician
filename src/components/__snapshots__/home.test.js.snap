// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a snapshot properly 1`] = `
<main
  className="container"
>
  <header>
    <nav>
      <a
        className="logo"
        href="/"
        onClick={[Function]}
      >
        <h1>
          Maths Magiciacian
        </h1>
      </a>
      <ul>
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            href="/quotes"
            onClick={[Function]}
          >
            Quotes
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <div
    className="page-layout"
  >
    <h1>
      Welcome to Our page!
    </h1>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam hendrerit nisi sed sollicitudin pellentesque. Nunc posuere purus rhoncus pulvinar aliquam. Ut aliquet tristique nisl vitae volutpat. Nulla aliquet porttitor venenatis. Donec a dui et dui fringilla consectetur id nec massa. Aliquam erat volutpat. Sed ut dui ut lacus dictum fermentum vel tincidunt neque. Sed sed lacinia lectus. Duis sit amet sodales felis. Duis nunc eros, mattis at dui ac, convallis semper risus. In adipiscing ultrices tellus, in susc ipit massa vehicula eu.”
    </p>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam hendrerit nisi sed sollicitudin pellentesque. Nunc posuere purus rhoncus pulvinar aliquam. Ut aliquet tristique nisl vitae volutpat. Nulla aliquet porttitor venenatis. Donec a dui et dui fringilla consectetu r id nec massa. Aliquam erat volutpat. Sed ut dui ut lacus dictum fermentum vel tincidunt neque. Sed sed lacinia lectus. Duis sit amet sodales felis. Duis nunc ero s, mattis at dui ac, convallis semper risus. In adipiscing ultrices tellus, in suscipit massa vehicula eu.”
    </p>
  </div>
</main>
`;
